openedx/core/djangoapps/zendesk_proxy/v0/views.py

```diff
@@ -0,0 +1,58 @@
+"""
+Define request handlers used by the zendesk_proxy djangoapp
+"""
+from rest_framework import status
+from rest_framework.parsers import JSONParser
+from rest_framework.response import Response
+from rest_framework.throttling import UserRateThrottle
+from rest_framework.views import APIView
+
+from openedx.core.djangoapps.zendesk_proxy.utils import create_zendesk_ticket
+
+ZENDESK_REQUESTS_PER_HOUR = 15
+
+
+class ZendeskProxyThrottle(UserRateThrottle):
+    """
+    Custom throttle rates for this particular endpoint's use case.
+    """
+    THROTTLE_RATES = {
+        'user': '{}/hour'.format(ZENDESK_REQUESTS_PER_HOUR),
```

Comment:
Should this be [anon](http://www.django-rest-framework.org/api-guide/throttling/#anonratethrottle) instead? It looks like using "user" will still throttled based on IP if not authenticated, but using "anon" seems more explicit. I could go either way!

Reply:
I just spoke with ops about this - we want a *global* throttle, not an IP-based one, as the attackers may be in possession of a large botnet with many addresses available. I'll work on adding such.

Reply:
Ah, ok! I think the `anon` solution works for that then?

Reply:
Not quite - [anon means unauthenticated requests get throttled while authenticated ones don't](https://github.com/edx/django-rest-framework/blob/master/rest_framework/throttling.py#L168)

It should be fairly easy to subclass `SimpleRateThrottle` to return a static value from `get_cache_key` though, that will do what we want.